src/solvers/dgmulti/dg.jl

```diff
@@ -76,41 +119,7 @@ function prolong2interfaces!(cache, u, mesh::AbstractMeshData, equations,
   StructArrays.foreachfield(mul_by!(rd.Vf), u_face_values, u)
 end
 
-function create_cache(mesh::VertexMappedMesh, equations, dg::DG,
-                      RealT, uEltype) where {DG <: Union{DGMultiWeakForm, DGMultiFluxDiff{<:SBP}}}
-
-  rd = dg.basis
-  md = mesh.md
-
-  # volume quadrature weights, volume interpolation matrix
-  @unpack wq, Vq = rd
-
-  # mass matrix, tuple of differentiation matrices
-  @unpack M, Drst, Pq = rd
-
-  # ∫f(u) * dv/dx_i = ∑_j (Vq*D_i)'*diagm(wq)*(rstxyzJ[i,j].*f(Vq*u))
-  weak_differentiation_matrices = map(D -> -M\((Vq*D)'*diagm(wq)), Drst)
-
-  # for use with flux differencing schemes
-  Qrst = map(D->Pq'*M*D*Pq, Drst)
-  Qrst_skew_Tr = map(A -> -0.5*(A-A'), Qrst) # Todo: simplices. Rename this in flux differencing PR
-
-  nvars = nvariables(equations)
-
-  # Todo: simplices. Factor common storage into a struct (MeshDataCache?) for reuse across solvers?
-  # storage for volume quadrature values, face quadrature values, flux values
-  u_values = StructArray{SVector{nvars, uEltype}}(ntuple(_->zeros(rd.Nq, md.num_elements), nvars))
-  u_face_values = StructArray{SVector{nvars, uEltype}}(ntuple(_->zeros(rd.Nfq, md.num_elements), nvars))
-  flux_face_values = similar(u_face_values)
-
-  # local storage for fluxes
-  local_values_threaded = [StructArray{SVector{nvars, uEltype}}(ntuple(_->zeros(rd.Nq), nvars)) for _ in 1:Threads.nthreads()]
-
-  return (; md, weak_differentiation_matrices, Qrst_skew_Tr, invJ = inv.(md.J),
-      u_values, local_values_threaded, u_face_values, flux_face_values)
-end
-
-function calc_volume_integral!(du, u::StructArray, volume_integral::VolumeIntegralWeakForm,
+function calc_volume_integral!(du, u, volume_integral::VolumeIntegralWeakForm,
                  mesh::VertexMappedMesh, equations, dg::DGMulti, cache)
```

Comment:
```suggestion
function calc_volume_integral!(du, u, volume_integral::VolumeIntegralWeakForm,
                               mesh::VertexMappedMesh, equations, dg::DGMulti, cache)
```

Reply:
Weird - I wonder why my formatting keeps doing this? I'm pretty sure I set tabs as 2 spaces in VSCode...